api/nakadi-event-bus-api.yaml

```diff
@@ -0,0 +1,1145 @@
+        rules in the order the rules are defined in the `EventType`.
+        No preexisting value might be changed (even if added by an enrichment rule). Violations on this will
+        force the immediate **rejection** of the Event with a 409 Conflict status. The invalid overwrite
+        attempt will be included in the respose's Problem object.
+
+        1. The incoming Event's relative ordering is evaluated according to the rule on the
+        `EventType`. Failure to evaluate the rule will **reject** the Event.
+        **Restriction**: POSTing Events based on its `EventType` requires it to have a
+        `PartitioningStrategy` defined. Violating this condition will produce a 405 Not allowed.
+
+      parameters:
+        - name: name
+          in: path
+          type: string
+          description: Name of the EventType
+          required: true
+      responses:
+        '200':
+          description: Ok
+        '400':
+          description: Bad request
+          schema:
+            $ref: '#/definitions/Problem'
+        '401':
+          description: Client is not authenticated
+          schema:
+            $ref: '#/definitions/Problem'
+        '404':
+          description: EventType not found.
+          schema:
+              $ref: '#/definitions/Problem'
+        '405':
+          description: Not allowed.
+          schema:
+              $ref: '#/definitions/Problem'
+        '409':
+          description: Conflict.
+          schema:
+              $ref: '#/definitions/Problem'
+        '422':
+          description: Unprocessable Entity
+          schema:
+            $ref: '#/definitions/Problem'
+        '500':
+          description: Server error
+          schema:
+            $ref: '#/definitions/Problem'
+
+  /topics:
+    get:
+      description: Lists all known `Topics` of this Nakadi cluster
+      tags:
+        - monitoring
+        - management-api
+      responses:
+        '200':
+          description: The request was successful. Returns a list `Topic`s in the response body.
+          schema:
+            type: array
+            description: An array of topics
+            items:
+              $ref: '#/definitions/Topic'
+        '401':
+          description: Client is not authenticated
+          schema:
+            $ref: '#/definitions/Problem'
+        '500':
+          description: Server error
+          schema:
+            $ref: '#/definitions/Problem'
+
+        '503':
+          description: Not available
+          schema:
+            $ref: '#/definitions/Problem'
+
+  '/topics/{topic}/events':
+    get:
+      tags:
+        - topic-api
+        - draft
+      description: |
+        Starts a stream delivery for the specified partitions of the given topic.
+
+        The event stream is formated as a sequence of `EventStreamBatch`es separated by `\n`. Each
+        `EventStreamBatch` constains a chunk of Events and a `Cursor` pointing to the end of the chunk (i.e.
```

Comment:
@aikej WebSockets and HTTP2 are very sexy indeed. And nakadi will support both these protocols in the future. Our goal now is to define a pure HTTH1.1 protocol.